client/extensions/wp-super-cache/preload-tab.jsx

```diff
@@ -30,28 +30,37 @@ import { isPreloadingCache } from './state/cache/selectors';
  */
 const renderCachePreloadInterval = ( {
 	handleChange,
-	isRequesting,
-	isSaving,
+	isDisabled,
 	preload_interval,
 } ) => (
 	<FormTextInput
 		className="wp-super-cache__preload-interval"
-		disabled={ isRequesting || isSaving }
+		disabled={ isDisabled }
 		min="0"
 		name="preload_interval"
 		onChange={ handleChange( 'preload_interval' ) }
 		step="1"
 		type="number"
-		value={ preload_interval || '' } />
+		value={ preload_interval || '0' } />
 );
 
 class PreloadTab extends Component {
-	state = {
-		preloadRefresh: true,
+	componentWillReceiveProps( nextProps ) {
+		this.state = {
+			preloadRefresh: parseInt( nextProps.fields.preload_interval, 10 ) === 0 ? false : true
```

Comment:
We do actually need state here. Because this is a derived setting, the HOC does not maintain the state for it as it does for other settings that are returned by the endpoint. If we removed the local state, then the toggle would not respond to user input at all.

I did make a small change in https://github.com/Automattic/wp-calypso/pull/14518/commits/80524a8018e9b1b9013a6d7f5e661e46c6d456a4 to set the initial state though.